sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -763,6 +763,30 @@ func withPrivilegedMounts() oci.SpecOpts {
 	})
 }
 
+// withStorageMounts adds options to share container storage mounts
+func withStorageMounts() oci.SpecOpts {
+	var mounts []runtimespec.Mount
+
+	storageDirs := []string{
+		"/var/lib/containerd",
+		"/var/lib/docker",
+		"/var/lib/kubelet",
+	}
```

Comment:
Should this be a constant, declared close to where it used?

Reply:
```golang
const storageDirs := []string{
                "/var/lib/containerd",
		"/var/lib/docker",
		"/var/lib/kubelet",
}

func withStorageMounts() oci.SpecOpts {
    var mounts []runtimespec.Mount

     for _, storageDir := range storageDirs {
          // ...
     }
}
```

Reply:
Go doesn't support `const` arrays or slices, so I don't think this alternative approach can work.